pkg/collector/scraper_test.go

```diff
@@ -51,6 +52,41 @@ func TestScraper(t *testing.T) {
 		}
 	}
 }
+func TestScraperAddsKubernetesClusterUUID(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log.Debug("metrics requested")
+		io.WriteString(w, testmetrics)
+	}))
+	defer ts.Close()
+
+	kubernetesClusterUUID := "kubernetes_cluster_uuid"
+	clusterUUID := "123-345-678000"
+	var kubernetesLabels []*dto.LabelPair
+	kubernetesLabels = append(kubernetesLabels, &dto.LabelPair{Name: &kubernetesClusterUUID, Value: &clusterUUID})
+	s, err := NewScraper("testscraper", ts.URL, kubernetesLabels, nil, 30*time.Second)
+	require.NoError(t, err)
+
+	ch := make(chan prometheus.Metric)
+	go s.Collect(ch)
+	for m := range ch {
+		if m.Desc().String() != `Desc{fqName: "testscraper_scrape_collector_success", help: "testscraper: Whether a collector succeeded.", constLabels: {}, variableLabels: [collector]}` &&
```

Comment:
nit: invert the if with `==` and use `continue` to prevent extra nesting

Reply:
fixed